### PR TITLE
Phase2-hgx325D Modify the validation codes of HGCalValidation in the test directory and make them useful only for D88 and beyond

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Modify the validation codes of HGCalValidation in the test directory and make them useful only for D88 and beyond

#### PR validation:

Use the runTheMatrix test workflows

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special;

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Validation/HGCalValidation/test/HGCGeometryCheck.cc [hunk 1]
```diff
@@ -121,8 +120,7 @@ void HGCGeometryCheck::analyze(const edm::Event &iEvent, const edm::EventSetup &
                                  << " Luminosity " << iEvent.luminosityBlock() << " Bunch " << iEvent.bunchCrossing();
 #endif
   //Accessing G4 information
-  edm::Handle<PHGCalValidInfo> infoLayer;
-  iEvent.getByToken(g4Token_, infoLayer);
+  const edm::Handle<PHGCalValidInfo> infoLayer = iEvent.getHandle(g4Token_);
```
Comment (rovere) [inline Validation/HGCalValidation/test/HGCGeometryCheck.cc:123]: Maybe missing a `&`?

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by emanueleusai** [comment]
+1

**Comment by emanueleusai** [comment]
type hgcal

**Comment by perrotta** [comment]
+1

**Pull request merged**